src/Validators/DependentRequiredFieldsValidator.php

```diff
@@ -104,49 +100,8 @@ protected function validateField($data, FieldList $fields, string $fieldName, ar
         return true;
     }
 
-    /**
-     * Ugly copypasta from SearchFilterable trait 'cause that's just easier.
-     * Don't use the trait directly 'cause if new methods are added to that trait, we don't want them here.
-     */
```

Comment:
I can't remember what the reasoning was there... but that trait hasn't been updated in a while so this seems fine to replace.